docs/40-troubleshooting.md

```diff
@@ -1,4 +1,64 @@
-# Getting logs without Azure Portal
+## Debugging and Troubleshooting
+
+### After creating my cluster, my nodes go to a 'startTaskFailed' state. Why?
+The most common case for this is that there was an issue with package installation or the custom script failed to run. To troubleshoot this you can simply download the output logs from the node.
+
+The following 2 nodes failed while running the start task:
```

Comment:
Just to be more verbose, but also a little clearer about the scenario:
"
Node ids are prepended with **tvm**. Lets say that when spinning up your cluster, the following 2 nodes failed while running the start task:
- tvm-769611554_1-20170912t183413z-p
- tvm-769611554_2-20170912t183413z-p

Here's how you would get the node ids and the logs from the nodes to debug:
"